src/it/tita/src/test/resources/log4j2-test.xml

```diff
@@ -13,7 +13,7 @@
 
         <!-- Main ones -->
         <AsyncLogger name="org.mule.maven.client" level="INFO"/>
-        <AsyncLogger name="org.mule.runtime.core.internal.processor.LoggerMessageProcessor" level="DEBUG"/>
+        <AsyncLogger name="org.mule.runtime.core.components.internal.processor.LoggerMessageProcessor" level="DEBUG"/>
```

Comment:
@acurci-at-mulesoft Is this something required for this test? If not... we should remove it. 
If this is needed, @mule-pachanady how are you going to handle backwards compatibility issues?
This is a test resource that may be used in any mule in connector's supported range. Are you going to port this change back to Mule 4.2.x?